### ISSUE TITLE
Feature: choose gamemode

### ISSUE THREAD

**Issue by blefeuvr** [blefeuvr/diep-rl#1]
For now, the gamemode is always set to sandbox

**Comment by Titaniumtown** [comment]
good idea, this is the best bot i've seen so far. I hope it gets better :)

**Comment by blefeuvr** [comment]
Hi there and thanks !
I won't be actively developing this project soon.
But feel free to make any PR !

**Comment by Titaniumtown** [comment]
I'm actively looking into the code and how I can improve it. I saw that in diep.py there is a self.gamemode which is set to 'sandbox' but changing it doesn't change what gamemode it selects.

**Comment by Titaniumtown** [comment]
haha, just noticed that you opened an issue on your own repo. I didn't see that at first.

**Comment by blefeuvr** [comment]
Yes, this value isn't used anywhere for now.
The change should be done [here](https://github.com/blefeuvr/diep-rl/blob/master/diep.py#L56)
The coordinates `(597, 58)` correspond to the **sandbox** selector button location
You could change these values depending on the value of `self.gamemode` to click on the right button.

But it will one other thing:
The detection of objects is made in [diep_injector.js](https://github.com/blefeuvr/diep-rl/blob/master/diep_injector/diep_injector.js) and is based on objects colour.

Since colors change between gamemodes, you will have to add colors of objects in the injector for any new gamemode.

**Comment by Titaniumtown** [comment]
oh ok, thanks :)
